tests/ops/fixtures/application_fixtures.py

```diff
@@ -592,6 +590,291 @@ def erasure_policy_two_rules(
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.gender").value,
+            "rule_id": user__gender_rule.id,
+        },
+    )
+
+    user__contact__address__city_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__address__city Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Masked"},
+            },
+        },
+    )
+    user__contact__address__city_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.city").value,
+            "rule_id": user__contact__address__city_rule.id,
+        },
+    )
+    user__contact__address__country_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__address__country Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Mars"},
+            },
+        },
+    )
+    user__contact__address__country_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.country").value,
+            "rule_id": user__contact__address__country_rule.id,
+        },
+    )
+    user__contact__email_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__email Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": f"{uuid4().hex}_MASKED@email.com"},
+            },
+        },
+    )
+    user__contact__email_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.email").value,
+            "rule_id": user__contact__email_rule.id,
+        },
+    )
+    user__contact__phone_number_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__phone_number Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "+12345678910"},
```

Comment:
Can we null out the number here too instead of writing a dummy value? There could be cases where the update fails from duplicate numbers.